src/System.CommandLine.Tests/ParserTests.cs

```diff
@@ -826,6 +826,33 @@ public void Commands_can_have_default_argument_values()
             GetValue(result, argument)
                   .Should()
                   .Be("default");
+
+            result.GetRequiredValue(argument)
+                  .Should()
+                  .Be("default");
+        }
+
+        [Fact]
+        public void GetRequiredValue_throws_when_argument_without_default_value_was_not_provided()
```

Comment:
`ParserTests` is a bit of a catchall. I think these would be better organized under `ArgumentTests` and `OptionTests`.